src/test/java/edu/ucsb/mapache/entities/SearchTests.java

```diff
@@ -0,0 +1,28 @@
+package edu.ucsb.mapache.entities;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import org.junit.jupiter.api.Test;
+
+public class StudentTests {
+    @Test
+  public void testSearch_getAndSetId() {
+    Search s = new Search();
+    s.setId(1);
+    assertEquals(1, s.getId());
+  }
+
+  @Test
+  public void testSearch_getAndSetSearchTerm() {
+    Search s = new Search();
+    s.setSearchTerm("term");
+    assertEquals("term", s.getSearchTerm());
+  }
+
+  @Test
+  public void testSearch_getAndSetCount() {
+    Search s = new Search();
+    s.setCount(123);
+    assertEquals(123, s.getCount());
+  }
+}
```

Comment:
Once you add the .equals, .toString, and .hashCode, you can consult this file for a good example of how to test a class like this one:

https://github.com/ucsb-cs156-w21/proj-ucsb-cs-las/blob/main/src/test/java/edu/ucsb/ucsbcslas/entities/TutorAssignmentTests.java

Note that you can test all of the getters and setters with one line of code:

```
@Test
  public void testGettersAndSetters() throws Exception {
    // See: https://github.com/codebox/javabean-tester
    JavaBeanTester.test(TutorAssignment.class);
  }
```